### DISCUSSION TITLE
'Remotely-save' plug-in for LogSeq

### DISCUSSION THREAD

**Discussion by SAI-GIT1** [remotely-save/remotely-save#392] [Polls]
POLL: 'Remotely-save' functionality for Obsidian should be a core feature. Porting this plug-in over to the community in LogSeq would be S-tier. Place your vote here!

**Comment by fyears** [comment]
the point is logseq doesn't support mobile plugin

**Reply by Slabhub** [reply]
it doesnt matter , for other platform, getting webdav support will be amazing.

**Reply by fyears** [reply]
does logseq have normail files like obsidian or have database like joplin? 

does it have official plugin market? any official plugin development doc?

and may i ask why don't you consider general solution like rclone?

**Reply by SAI-GIT1** [reply]
> does logseq have normail files like obsidian or have database like joplin? 
Logseq files are in Markdown format using basic filesystem folder hierarchy.


> does it have official plugin market? any official plugin development doc?
Plugins for desktop, not for mobile yet.
https://docs.logseq.com/?ref=blog.logseq.com#/page/plugins%20101


> and may i ask why don't you consider general solution like rclone?
Mobile shared platform primary use case. iPhone, iPad to Desktop(Mac/Windows) is my complete workflow. Dropbox, i.e. Remotely-Sync plugin using Obsidian with iOS and Windows Desktop would be ideal.

**Reply by fyears** [reply]
but as you said, logseq doesn't support mobile plugin yet. so remotely save for logseq doesn't help mobile-desktop workflow...

**Reply by SAI-GIT1** [reply]
> but as you said, logseq doesn't support mobile plugin yet. so remotely save for logseq doesn't help mobile-desktop workflow...

Building the plugin for desktop would be a promisingly good start.

**Comment by Slabhub** [comment]
> does logseq have normail files like obsidian or have database like joplin?
> logseq use md files like obsidian but has 3 folder system  (journal folder, pages folder whiteboards folder) . They will release a "DB version" but this version is based sqllite and very adapted MD files system . https://github.com/logseq/logseq/blob/master/deps/db/src/logseq/db/schema.cljs
> does it have official plugin market? any official plugin development doc?
> https://github.com/logseq/plugins
> and may i ask why don't you consider general solution like rclone?
actually  remotely save plugin will be perfect bridge for obsidian mobile usage to logseq desktop platforms. Because we can use obsidian and logseq together . So this usage will be perfect solution for absent of logseq mobile plugin support. 
https://hub.logseq.com/integrations/aV9AgETypcPcf8avYcHXQT/how-to-use-obsidian-and-logseq-together-and-why-markdown-matters/1rqp92wgow7wGXS37Ckz1U
